.github/workflows/release.yml

```diff
@@ -39,9 +39,7 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         with:
-          publish:
-            - cp README.md packages/wrangler/README.md
```

Comment:
There is a copy command in the prerelease too: https://github.com/cloudflare/wrangler2/blob/485e0dcf9966b03c641cdbf38c6c626e6469d1c1/.github/workflows/prereleases.yml#L35-L36